airbyte-cdk/python/airbyte_cdk/sources/declarative/declarative_component_schema.yaml

```diff
@@ -2153,6 +2153,14 @@ interpolation:
       examples:
         - start_date: 2010-01-01
           api_key: "*****"
+    - title: parameters
+      description: Additional runtime parameters, to be used for string interpolation, can be set specifically for each stream.
```

Comment:
```suggestion
      description: Additional runtime parameters, to be used for string interpolation. Parameters can be passed down from a parent component to its subcomponents using the $parameters key. This can be used to avoid repetitions.
```